krcal/map_builder/map_builder_functions.py

```diff
@@ -738,4 +747,8 @@ def map_builder(config):
     write_complete_maps(asm      = final_map          ,
                         filename = config.file_out_map)
     print("Map successfully computed and saved in : {0}".format(config.file_out_map))
-    print("Control histograms saved in            : {0}".format(config.file_out_hists))
+    if monitoring:
+        par_selection_to_plot_vs_time(evol_table = final_map.t_evol,
+                                      file_name  = monitoring              )
```

Comment:
this last parenthesis should learn how to get on well with his friends and come closer